libkbfs/block_journal.go

```diff
@@ -725,39 +754,49 @@ func (be blockEntriesToFlush) flushNeeded() bool {
 // Only entries with ordinals less than the given ordinal (assumed to
 // be <= latest ordinal + 1) are returned.
 func (j *blockJournal) getNextEntriesToFlush(
-	ctx context.Context, end journalOrdinal) (
-	entries blockEntriesToFlush, err error) {
+	ctx context.Context, end journalOrdinal, maxToFlush int) (
+	entries blockEntriesToFlush, maxMDRevToFlush MetadataRevision, err error) {
 	first, err := j.j.readEarliestOrdinal()
 	if os.IsNotExist(err) {
-		return blockEntriesToFlush{}, nil
+		return blockEntriesToFlush{}, MetadataRevisionUninitialized, nil
```

Comment:
Ah, never mind I see that you handle it in the caller. Probably worth a comment here, though.